lms/static/scripts/frontend_apps/components/FilePickerApp.tsx

```diff
@@ -69,6 +70,35 @@ function contentDescription(content: Content) {
   }
 }
 
+/**
+ * Fetch additional configuration needed by the file picker app.
```

Comment:
It could do, although putting this function here co-locates the function which fetches the data / configuration needed by a view, with the view that uses it, which I think makes sense as the two are liable to change together.